tests/json.rs

```diff
@@ -2,6 +2,23 @@ use slack_blocks::block_elements::BlockElement;
 use slack_blocks::blocks::Block;
 use slack_blocks::compose;
 
+fn check_no_nulls(json: &serde_json::Value) -> Result<(), ()> {
```

Comment:
I wonder if it would be easy to do this recursively; for example if i pass
```rust
json!({
    "obj": {
        "uhOh": null
    }
})
```
to this function, it would return an `Ok(())` yeah? It would be _(a)_ nice _(to have)_ if it checked deeply into structures

Reply:
I'm stupid. it is recursive lol